app/assets/stylesheets/application.css

```diff
@@ -103,3 +103,50 @@ button {
   justify-content: space-between;
   align-items: center;
 }
+
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you add your own logo (Not Microverse logo) at the top of your readme file to make yourself a brand in the tech industry to catch the eyes of recruiters 👍